### ISSUE TITLE
make FEP schedules configurable on a per solvent basis

### ISSUE THREAD

**Issue by orbeckst** [Becksteinlab/MDPOW#2]
Requires #1 to be implemented.

**Comment by ianmkenney** [comment]
@orbeckst  Is this to say that if no values are provided in the configuration file, we are to use the defaults in the Goct subclass?

**Comment by orbeckst** [comment]
Yes, we should have some defaults and the Goct ones are as good as any (at least they work for octanol).

I would just call them "non-water solvent" defaults and it would be good to have them explicit in the config file somewhere.

Or perhaps better: Raise an error if no solvent parameters are given and we ship config files with explicit FEP schedules for water, octanol and cyclohexane.
